configs/rk3399-vaaman_defconfig

```diff
@@ -110,6 +110,7 @@ CONFIG_IO_DOMAIN=y
 CONFIG_ROCKCHIP_IO_DOMAIN=y
 CONFIG_DM_PMIC=y
 CONFIG_PMIC_RK8XX=y
+CONFIG_REGULATOR_FAN53555=y
```

Comment:
Why is this needed? We don't have any fans on the board as of now!

Reply:
Okay, I will remove this. However, it was initially included because we have the option to add a fan, and that's why it was included in the first place.

Reply:
okay, the fan can be started through kernel itself, on time of u-boot that is not required, if that requires, we will add that in future.

Reply:
change this, we'll merge

Reply:
okay!